dune/harmonizer/dunesql/dunesql.py

```diff
@@ -0,0 +1,58 @@
+from sqlglot import TokenType, exp, transforms
+from sqlglot.dialects.trino import Trino
+
+from dune.harmonizer.dunesql.transform import (
+    cast_boolean_strings,
+    cast_date_strings,
+    concat_of_hex_string_to_bytearray_concat,
+    pipe_of_hex_strings_to_bytearray_concat,
+    remove_lower_around_hex_strings,
+    rename_bytea2numeric_to_bytearray_to_bigint,
+    replace_0x_strings_with_hex_strings,
+)
+
+
+class DuneSQL(Trino):
+    """The DuneSQL dialect is the dialect used to execute SQL queries on Dune's crypto data sets
+
+    DuneSQL is the Trino dialect with slight modifications."""
+
+    class Tokenizer(Trino.Tokenizer):
+        """Text -> Tokens"""
+
+        HEX_STRINGS = ["0x", ("X'", "'")]
+        KEYWORDS = Trino.Tokenizer.KEYWORDS | {
+            "UINT256": TokenType.UINT256,
+            "INT256": TokenType.INT256,
+        }
+
+    class Parser(Trino.Parser):
+        """Tokens -> AST"""
+
+        pass
```

Comment:
We were adding `(U)INT256` tokens here before, but those are already present in the `Trino.Parser`.